Procfile

```diff
@@ -1 +1,2 @@
+release: ./release-tasks.sh
```

Comment:
It looks like this file isn't executable, so I think this might throw an error. You should be able to `chmod +x release-tasks.sh` and commit the permissions change.

Reply:
@rlgreen91 it looks like you didn't make `release-tasks.sh` executable. I think the next time you deploy it will throw an error. Just FYI.